app/data/jsonSchemaForm/fundingParameterEPUiSchema.ts

```diff
@@ -98,6 +128,9 @@ export const fundingParameterEPUiSchema = {
       amount: {
         "ui:title": `Additional Funding Amount`,
         "ui:widget": "NumberWidget",
+        "ui:tooltip": {
+          text: "<div><ul><li>The amount of payment from Funding Source N other than CIF.</li><li>Typically found in the original proposal (solicitation folder of the LAN).</li></ul></div>",
+        },
```

Comment:
`solicitation folder on the LAN`? based on the Word document?!

Reply:
fixed!